OpenImis.ModulesV2/PolicyModule/Helpers/OutputParameter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Text;
+
+namespace OpenImis.ModulesV2.PolicyModule.Helpers
+{
+    public static class OutputParameter
```

Comment:
I see this class in multiple Modules. To remove duplicates, can you move it to Utils and use it from there?

Reply:
It will be in another task